### PR TITLE
Install and ingest doc mods.

### PR DESCRIPTION
**Pull request by DavidBrownlee**
Documentation modification resulting from my experience of installing CanDIGv2 on Ubuntu.  Mostly changes to the docker install, but a few corrections as well.

### CODE REVIEW

File: docs/ingest-and-test.md [hunk 1]
```diff
@@ -6,11 +6,27 @@ These instructions will lead you through some basic functionality tests, ingesti
...
 make test-integration
 ```
 
+If the 4 federation tests fail, restart the federation container with:
+```bash
+make clean-federation
+make build-federation
+make compose-federation
+```
+
```
Comment (OrdiNeu) [inline docs/ingest-and-test.md:26]:
```suggestion
```

File: etc/tests/tokenTest.sh [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```
Comment (OrdiNeu) [inline etc/tests/tokenTest.sh:1]: While this is useful, I'm not sure that tests/ is the right folder for this. After a bit of discussion, Daisie and I found that there's no real spot for us to put helper scripts like this (that aren't part of the build process), so... I think here is fine until we reorganize it.

**Review by OrdiNeu — APPROVED** [review]

**Comment by DavidBrownlee** [comment]
Thanks for the review.  I agree this isn't a great location for tokenTest.sh, but this is only a modification of a previous script (that was in the same location, but is now absent).  The new script pulls values from etc/secrets/... directly so it can be run from the command line without any copy & paste operations.

**Comment by mshadbolt** [comment]
Is this ready to merge in? We have another PR open with doc changes so would be good to merge this in, then resolve any conflicts and merge the other one too

**Pull request merged**